### ISSUE TITLE
Clippy: `error: boolean expression will never evaluate to 'true'`

### ISSUE THREAD

**Issue by virtualritz** [Roughsketch/imagesize#35]
```
error: boolean expression will never evaluate to 'true'
  --> src/formats/pnm.rs:59:8
   |
59 |     if header[1] < b'1' && header[1] > b'6' {
   |        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: since `b'1'` < `b'6'`, the expression evaluates to false for any value of `header[1]`
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#impossible_comparisons
   = note: `#[deny(clippy::impossible_comparisons)]` on by default
```

**Comment by virtualritz** [comment]
Probably you meant `||` instead of `&&` here?

**Comment by Roughsketch** [comment]
Fix was included in #34
